src/core/Metadata.cpp

```diff
@@ -28,6 +28,7 @@
 
 const int Metadata::DefaultHistoryMaxItems = 10;
 const int Metadata::DefaultHistoryMaxSize = 6 * 1024 * 1024;
+const int Metadata::DefaultAutosaveDelay = 0;
```

Comment:
I would name as `DefaultAutosaveDelaySec`.

Reply:
Fixed in [3a1e023](https://github.com/keepassxreboot/keepassxc/pull/9100/commits/3a1e0231d88af17d161555fa7182d6b9166b162a)